btcutil/psbt/psbt.go

```diff
@@ -395,3 +396,20 @@ func (p *Packet) SanityCheck() error {
 
 	return nil
 }
+
+// GetTxFee returns the transaction fee.  An error is returned if a transaction
+// input does not contain any UTXO information.
+func (p *Packet) GetTxFee() (btcutil.Amount, error) {
+	sumInputs, err := SumUtxoInputValues(p)
+	if err != nil {
+		return btcutil.Amount(0), err
```

Comment:
You can just do `return 0, err`. Since 0 is then a constant, that's cast automatically at compile time. Makes it a bit easier to read.